src/index.html

```diff
@@ -16,7 +16,15 @@
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Stop watch</h1>
+  <body class="page-body">
+    <div class="stopwatch">
```

Comment:
```suggestion
    <div class="stopwatch stopwatch--speed-up">
```
we need that modifier here